app/src/main/java/com/framgia/forder/screen/shopmanagement/ShopManagementContract.java

```diff
@@ -1,7 +1,11 @@
 package com.framgia.forder.screen.shopmanagement;
 
+import com.framgia.forder.data.model.Shop;
```

Comment:
ERROR: Unused import: com.framgia.forder.data.model.Shop. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports